baseapp/sidechannel_baseapp.go

```diff
@@ -0,0 +1,93 @@
+package baseapp
+
```

Comment:
we don't need `baseapp` suffix for the filename may be. It's already inside the baseapp package

Reply:
Fixed.